### PR TITLE
fix: disable prefetch for anvil tests

### PR DESCRIPTION
**Pull request by DanielSchiavini**
### What I did
Fixed the integration tests

### How I did it
Disabling prefetching in anvil integration tests

### How to verify it
Tests should pass

### Description for the changelog
N/A

### Cute Animal Picture
image

### CODE REVIEW

**Comment by DanielSchiavini** [comment]
Tests are green on my fork
https://github.com/DanielSchiavini/titanoboa/actions/runs/7740967856

File: tests/integration/network/anvil/test_network_env.py [hunk 1]
```diff
@@ -56,4 +56,20 @@ def test_raise_exception(simple_contract, t):
         simple_contract.raise_exception(t)
 
 
+# Prefetch is disabled currently since anvil doesn't support `prestateTracer`.
+# This test will fail when anvil is fixed, so we can re-enable prefetching.
+# See fixture `anvil_env`
+def test_debug_traceCall_tracer_ignored(simple_contract):
+    assert simple_contract.totalSupply() == STARTING_SUPPLY
+
+    boa.env._fork_try_prefetch_state = True
```
Comment (charles-cooper) [inline tests/integration/network/anvil/test_network_env.py:65]: `tmp = boa.env._fork_try_prefetch_state` so we can restore it later

File: tests/integration/network/anvil/test_network_env.py [hunk 2]
```diff
@@ -56,4 +56,20 @@ def test_raise_exception(simple_contract, t):
...
+    try:
+        with pytest.raises(ValueError) as excinfo:
+            boa.loads(code, STARTING_SUPPLY)
+        expected = "when sending a str, it must be a hex string. Got: 'failed'"
+        assert expected == str(excinfo.value)
+    finally:
+        boa.env._fork_try_prefetch_state = False
```
Comment (charles-cooper) [inline tests/integration/network/anvil/test_network_env.py:72]:
```suggestion
        boa.env._fork_try_prefetch_state = tmp
```

**Review by charles-cooper — APPROVED** [review]
lgtm. thanks!

**Pull request merged**